Hardware/Firmware/mainboard/README.md

```diff
@@ -0,0 +1,18 @@
+# Mainboard firmware
+
+## Build flags
```

Comment:
Please add a short explanation on how to set these flags. I always forget how to set them anyway, and looking here would be much quicker than going through PlatformIO docs.

Reply:
And a list of available environments and an explanation to set those would be appreciated as well.